### PR TITLE
Create FCCOSEth002.xml

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: CairoFCCOS/FCCOSEth002.xml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="page" quantity="230">230</measure>
+                                    <dimensions type="outer" unit="cm">
```
Comment (eu-genia) [inline CairoFCCOS/FCCOSEth002.xml:98]: we once said it is desirable to use mm

File: CairoFCCOS/FCCOSEth002.xml [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+                                        <height>20.5</height>
+                                        <width>18</width>
+                                    </dimensions>
+                                </extent>
+                                <condition key="deficient">The binding is broken.</condition>
```
Comment (eu-genia) [inline CairoFCCOS/FCCOSEth002.xml:103]: also repeated under binding below? needed twice?

**Review by eu-genia — APPROVED** [review]

File: CairoFCCOS/FCCOSEth002.xml (hunk 2 above)
Comment (thea-m) [reply CairoFCCOS/FCCOSEth002.xml:103]: I agree it is redundant. The reason I encoded it in this way is that this one sentence is the only information we have on the actual presence of a binding, and it equally applies to the condition. What would you propose?

File: CairoFCCOS/FCCOSEth002.xml (hunk 1 above)
Comment (thea-m) [reply CairoFCCOS/FCCOSEth002.xml:98]: The reason I didn't convert them is that I suspect that whole numbers given in centimeters might be rounded up, and converting into mm would simulate more preciseness than might be present here. On the other hand, if the filtering and searching benefits for quantitative analyses would outweigh that, let's do it.

File: CairoFCCOS/FCCOSEth002.xml (hunk 2 above)
Comment (eu-genia) [reply CairoFCCOS/FCCOSEth002.xml:103]: Am not sure, lets leave it

**Pull request merged**